zaza/openstack/charm_tests/ceph/rbd_mirror/tests.py

```diff
@@ -197,9 +198,14 @@ def get_pools(self):
                 'ceph-mon' + self.site_b_app_suffix,
                 model_name=self.site_b_model),
             model_name=self.site_b_model)
-        return sorted(site_a_pools.keys()), sorted(site_b_pools.keys())
-
-    def get_failover_pools(self):
+        site_a_pools = set(site_a_pools.keys())
+        site_b_pools = set(site_b_pools.keys())
+        if not include_internal_pools:
+            site_a_pools -= INTERNAL_POOLS
+            site_b_pools -= INTERNAL_POOLS
+        return list(site_a_pools), list(site_b_pools)
```

Comment:
I don't know if it matters, but this is now returns both a different type (a `Tuple[List, List]` rather than `Tuple[Iterator, Iterator]`) and they are no longer sorted?

Reply:
The function did return lists (calling `sorted` on a dict's keys returns a mutable list), and the call to `sorted` was unneeded.